app/controllers/ProfileViewsStreamController.scala

```diff
@@ -1,27 +1,25 @@
 package controllers
 
-import play.api.mvc.{Action, Controller}
 import data.ServiceClient
-import ui.{HtmlStream, Pagelet}
-import ui.HtmlStreamImplicits._
 import play.api.libs.concurrent.Execution.Implicits._
+import ui.HtmlStreamImplicits._
+import ui.{HtmlStream, Pagelet}
 
 /**
  * An example standalone module that represents the "Who's Viewed Your Profile" module from the LinkedIn homepage.
- * It's identical to the Wvyp controller, except that the data is streamed to the browser as soon as it's available in
+ * It's identical to the ProfileViewsController controller, except that the data is streamed to the browser as soon as it's available in
  * small "pagelets", similar to Facebook's BigPipe.
  */
-object WvypStream extends Controller {
+object ProfileViewsStreamController {
+
+  def stream = HtmlStream {
```

Comment:
Good point, I changed to be Controller now and can be served as a page
```scala
object UpdateViewsStreamController  extends ChunkedController {

  override def stream = HtmlStream {

    val likesCountFuture = ServiceClient.makeServiceCall("likes")
    val commentsCountFuture = ServiceClient.makeServiceCall("comments")

    val likesStream =Pagelet.renderStream(likesCountFuture.map(str => views.html.pagelets.updates.likesCount(str.toInt)), "likeCount")
    HtmlStream.interleave(likesStream)
  }

  override def holder(): HtmlStream = views.stream.pagelets.updates.updates()
}
```

My concern was, and correct me if I'm wrong, that the composed page was doing the REST calls and not calling the pagelet controller, so we ended up writing 2 controllers instead.